src/drivers/actuators/voxl_esc/voxl_esc.cpp

```diff
@@ -417,11 +441,11 @@ int VoxlEsc::parse_response(uint8_t *buf, uint8_t len, bool print_feedback)
 			switch (ret) {
 			case ESC_ERROR_BAD_CHECKSUM:
 				_rx_crc_error_count++;
-				//PX4_INFO("BAD ESC packet checksum");
+				PX4_INFO("BAD ESC packet checksum");
```

Comment:
These 2 prints can go back to being commented out?

Reply:
I will comment them back out